### PR TITLE
Update env version and fix user_column_names_param

### PR DESCRIPTION
**Pull request by aggarwal-k**

### CODE REVIEW

**Review by jpmann — APPROVED** [review]

File: assets/training/finetune_acft_hf_nlp/components_v2/pipeline_components/text_generation/spec.yaml [hunk 1]
```diff
@@ -141,6 +141,7 @@ jobs:
       train_file_path: '${{parent.inputs.train_file_path}}'
       validation_file_path: '${{parent.inputs.validation_file_path}}'
       system_properties: '${{parent.inputs.system_properties}}'
+      user_column_names: '${{parent.inputs.text_key}},${{parent.inputs.ground_truth_key}}'
```
Comment (amangupta26) [inline assets/training/finetune_acft_hf_nlp/components_v2/pipeline_components/text_generation/spec.yaml:144]: Why this change is needed?

Comment (aggarwal-k) [reply assets/training/finetune_acft_hf_nlp/components_v2/pipeline_components/text_generation/spec.yaml:144]: Accidentally got removed in previous PR, readding it now

**Review by amangupta26 — APPROVED** [review]
